uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/ssp-la-manager.json

```diff
@@ -10,7 +10,8 @@
       "label": "SSP / Manager Leave / Legend expanded",
       "url": "{{siteUrl}}/manager-leave#/manager-leave/calendar",
       "onReadyScript": "ssp/manager-leave/calendar-legend-expanded.js",
-      "user": "civihr_manager"
+      "user": "civihr_manager",
+      "requireSameDimensions": false
```

Comment:
i'll add it, but the short answer to "why" is "i don't know" unfortunately. For those couple scenarios backstopjs sometimes, at random, takes screenshots with different dimensions (i think you also saw it first-hand when testing), so we have to tell it to ignore them during the comparison, otherwise we'll end up with false positives

Reply:
Added a new "Avoid dimensions-based false positives" section in the PR description